addon-test-support/ember-qunit/test-isolation-validation.js

```diff
@@ -1,7 +1,8 @@
+/* eslint-disable no-console */
 import QUnit from 'qunit';
 import { run } from '@ember/runloop';
 import { waitUntil, isSettled, getSettledState } from '@ember/test-helpers';
-import TestDebugInfo, { getDebugInfo } from './-internal/test-debug-info';
+import { getDebugInfo } from '@ember/test-helpers/-internal/debug-info';
```

Comment:
I'd like to remove this also so that ember-qunit _only_ uses public APIs (aka things exported from `@ember/test-helpers` itself).

We should either make `getSettledState().debugInfo` be `undefined` when `getDebugInfo()` is not available, so that we can swap to using `getSettledState().debugInfo !== undefined`, or just expose the two helper functions at the index.

What do you think?

Reply:
I also don't like this. I didn't do a good job anticipating the public needs when I made my `@ember/test-helpers` change. I'll fix this now.

I'm partial to just having a public export. Does that sound ok to you?

Reply:
Implemented in https://github.com/emberjs/ember-test-helpers/pull/575